test/test_config.h.in

```diff
@@ -48,4 +51,110 @@ void unsetenv(const char * name)
   return;
 }
 #endif
+
+namespace sdf
+{
+  namespace testing
+  {
+    /// \brief Method to retrive root directory of project source
+    ///
+    /// This is used to get various test files
+    /// \param[inout] _dataDir Full path to the source directory
+    /// \return True if directory is set correctly, false otherwise
+    bool ProjectSourcePath(std::string &_sourceDir)
+    {
+      // Bazel builds set TEST_SRCDIR
+      if(char* dataDir = std::getenv("TEST_SRCDIR"))
```

Comment:
windows CI doesn't like this `getenv` call

~~~
'getenv': This function or variable may be unsafe. Consider using _dupenv_s instead. 
To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
~~~

https://build.osrfoundation.org/job/sdformat-ci-pr_any-windows7-amd64/1825/msbuild/